api/lib/opentelemetry/trace/span.rb

```diff
@@ -78,6 +78,28 @@ def add_attributes(attributes)
         self
       end
 
+      # Add a link to a {Span}.
+      #
+      # Adding links at span creation using the `links` option is preferred
+      # to calling add_link later, because head sampling decisions can only
+      # consider information present during span creation.
+      #
+      # Example>
```

Comment:
Using a colon matches the documentation style for `Span#add_event`:
https://github.com/open-telemetry/opentelemetry-ruby/blob/6623a3e41534622fb30bd30ec7892ea2e1b4a6b8/api/lib/opentelemetry/trace/span.rb#L81-L85

```suggestion
      # Example:
```